element-connector/pom.xml

```diff
@@ -21,11 +22,23 @@
 			<artifactId>junit</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<!-- https://mvnrepository.com/artifact/org.hamcrest/hamcrest-library -->
+		<dependency>
+			<groupId>org.hamcrest</groupId>
+			<artifactId>hamcrest-library</artifactId>
+			<version>1.3</version>
```

Comment:
I would like to add this dependency to the parent POM's managed dependencies and omit the version here. This library can then be used all other modules as well without the need to supply the version.